lte/gateway/python/magma/redirectd/BUILD.bazel

```diff
@@ -41,19 +41,23 @@ py_binary(
 
 py_library(
     name = "redirectd_lib",
-    srcs = [
-        "redirect_server.py",
-        "redirect_store.py",
-    ],
+    srcs = ["redirect_server.py"],
     data = [
         "templates/404.html",
         "templates/layout.html",
     ],
     visibility = ["//visibility:public"],
     deps = [
```

Comment:
The deps
```
 "//lte/protos:policydb_python_proto",
 "//orc8r/gateway/python/magma/common/redis:client",
```
 should be moved to `redirect_store`.

Reply:
Fixed.